src/msg.rs

```diff
@@ -284,6 +294,37 @@ impl Validate for ExecuteMsg {
                     }
                 }
             }
+            ExecuteMsg::ModifyContract {
+                approvers: _,
+                executors: _,
+                ask_fee_rate,
+                ask_fee_account,
+                bid_fee_rate,
+                bid_fee_account,
+                ask_required_attributes: _,
+                bid_required_attributes: _,
+            } => {
+                match (ask_fee_rate, ask_fee_account) {
```

Comment:
So the Modify has to include both rate + account? would it matter if the rate was changed but no bid/ask fee account was associated with it since there should already be one?

Reply:
oh, if the fees had never been set, then you would need both